Swift/Scope.swift

```diff
@@ -59,5 +59,5 @@ public final class Scope {
     }
     
     let _impl: CBLScope
-    weak var _db: Database?
+    let _db: Database?
```

Comment:
Is there the case the database is nil?

Reply:
no need, it was from the previous weak reference.